### DISCUSSION TITLE
Saving Game Just Like Stock Miyoo OS

### DISCUSSION THREAD

**Discussion by unbrokenblues** [OnionUI/Onion#1127] [Ideas]
I love Onion OS however I really dislike the way game switcher works. I really prefer the way Stock OS save the game which is: Menu > Save Slot and Choose the slot. All done. No fuss. 
I like to save my game on specific times on my games and I like to choose the save game slots myself.
Is there any way to use Onion OS but get the same Menu (Save game) Feature as the stock OS?

**Comment by Phanturian** [comment]
I don't think the stock save menu is going to be reimplemented considering Onion aims to streamline the user experience. You can still press Menu + Select during a game to bring up the Retroarch menu, scroll down and enter Save States, and then choose your save slots and save/load from there. The functionality you want is still there, you just don't get the same screen you like from the Stock OS.
